google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsAttemptCallable.java

```diff
@@ -116,25 +116,16 @@ public Object getTransportCode() {
   @Nonnull private TimedAttemptSettings attemptSettings;
 
   // Parent controller
-  private RetryingFuture<Void> externalFuture;
+  private RetryingFuture<MutateRowsAttemptResult> externalFuture;
 
   // Simple wrappers for handling result futures
-  private final ApiFunction<List<MutateRowsResponse>, Void> attemptSuccessfulCallback =
-      new ApiFunction<List<MutateRowsResponse>, Void>() {
-        @Override
-        public Void apply(List<MutateRowsResponse> responses) {
-          handleAttemptSuccess(responses);
-          return null;
-        }
-      };
+  private final ApiFunction<List<MutateRowsResponse>, MutateRowsAttemptResult>
+      attemptSuccessfulCallback = responses -> handleAttemptSuccess(responses);
```

Comment:
Please revert extra formatting changes...it makes it hard to see the real change

Reply:
Done